docker-compose.yml

```diff
@@ -538,7 +538,7 @@ services:
       - ./data/files:/share
     environment:
       ALLOW_SUPER_CONSUMER: "true"
-      ALLOWED_ACCOUNTS: "http://services.lblod.info/diff-consumer/account,http://data.lblod.info/foaf/account/id/dd85f516-ee7c-406b-8245-91ce7f9545b7"
+      ALLOWED_ACCOUNTS: "http://services.lblod.info/diff-consumer/account,http://data.lblod.info/foaf/account/id/dd85f516-ee7c-406b-8245-91ce7f9545b7,http://data.lblod.info/foaf/account/id/4171b7a5-a4d6-42d0-bf37-f97b135fb885"
```

Comment:
Where did `http://data.lblod.info/foaf/account/id/4171b7a5-a4d6-42d0-bf37-f97b135fb885` come from? Just wondering since they are of type `OnlineAccount` but don't have any other properties (when inspecting the production database).

Reply:
From felix -> https://binnenland.atlassian.net/browse/DGS-18?focusedCommentId=86870

Reply:
I see the line got changed with a PR, but he pointed to https://github.com/lblod/app-digitaal-loket/blob/master/docker-compose.yml#L857

Reply:
Ah perfect then; it's clear now.